pkg/reconciler/route/resources/ingress.go

```diff
@@ -95,10 +95,10 @@ func MakeIngress(
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      names.Ingress(r),
 			Namespace: r.Namespace,
-			Labels: map[string]string{
+			Labels: resources.UnionMaps(r.GetLabels(), map[string]string{
```

Comment:
Nit: to comply with the code below, can we use `r.ObjectMeta.Labels` here? Or even prettier: Change below to be `r.GetAnnotations()`?

Reply:
Yes, it should be consistent. I changed to use `r.ObjectMeta.Labels`.